roles/openshift_metrics/vars/main.yaml

```diff
@@ -18,3 +18,4 @@ hawkular_persistence: "{% if openshift.hosted.metrics.storage.kind != None %}tru
 hawkular_type: "{{ 'origin' if deployment_type == 'origin' else 'enterprise' }}"
 
 metrics_upgrade: openshift.hosted.metrics.upgrade | default(False)
+openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
```

Comment:
Not sure, I don't think either are required by openshift_metrics, and if something is defined in a roles vars.yml, am I correct that it's only valid there within that role and undefined elsewhere?

In general it looks like roles that need this define it themselves.